docs/Intro.md

```diff
@@ -20,9 +20,9 @@ So go do it! Go! Go [make an app](https://discord.com/developers/applications) a
 
 We love games, and we love helping game developers. If YOU have a game that you want to supercharge with Discord, then we have just the thing for you.
 
-#### Your Game On Discord
+#### Your Game With Discord
```

Comment:
This is a change in wording semantics that isn't really related to removing the game sales reference. I'd recommend restoring it and moving this to a separate issue/PR.